<?xml version="1.0"?>
<!-- 
  Wrapper launch file for the i5 specific robot_state node.
-->
<launch>
  <!-- IP of robot (or PC running simulation) -->
  <arg name="robot_ip" doc="IP of controller" />

  <node  name="io_service" 
    pkg="i5_driver" type="io_service">
    <param name="robot_ip_address" type="str" value="$(arg robot_ip)" />
    <!-- <param name="port" type="int" value=11003 /> -->
  </node>
</launch>